app/org/sagebionetworks/bridge/play/controllers/ParticipantController.java

```diff
@@ -95,8 +96,24 @@ public Result updateSelfParticipant() throws Exception {
     public Result getParticipants(String offsetByString, String pageSizeString, String emailFilter,
             String startDateString, String endDateString) {
         UserSession session = getAuthenticatedSession(RESEARCHER);
-        
         Study study = studyService.getStudy(session.getStudyIdentifier());
+        
+        return getParticipantsInternal(study, offsetByString, pageSizeString, emailFilter, startDateString,
+                endDateString);
+    }
+
+    public Result getParticipantsForWorker(String studyId, String offsetByString, String pageSizeString, String emailFilter,
+            String startDateString, String endDateString) {
+        getAuthenticatedSession(WORKER);
```

Comment:
The key difference of this method is that we get the study from a parameter, and not from the current user's session. And we don't want researchers to be able to retrieve content in another person's study.